android/src/main/java/com/imagepicker/ImagePickerModule.java

```diff
@@ -438,6 +438,10 @@ public void onActivityResult(Activity activity, int requestCode, int resultCode,
       }
       else
       {
+        // Newly captured images should only leave one copy on disk.
+        if (requestCode == REQUEST_LAUNCH_IMAGE_CAPTURE) { 
```

Comment:
I agree with this Pull request, but I think that it should be located into MediaUtils as static method.
Please move this logic to `MediaUtils.removeOriginalIfNeeded(imageConfig, requestCode)` and use it at ImagePickerModule.